front-end/tick-it/src/components/VenueListItem.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import { useNavigate, useParams } from 'react-router-dom'
+
+export default function VenueListItem (props) {
+    const [location, setLocation] = useState([])
+
+    useEffect(() => {
+        try {
+            const response = axios.get(`localhost:8000/locations`)
```

Comment:
You do not need to make an API call here. This is just the blueprint for information